src/satellite_consumer/run.py

```diff
@@ -28,59 +23,86 @@
 except PackageNotFoundError:
     __version__ = "v?"
 
+# Initialize Sentry
+
+
+sentry_sdk.init(
+    dsn="https://12927b5f211046b575ee51fd8b1ac34f@o1.ingest.sentry.io/1",  
+    traces_sample_rate=1.0,  
+    environment="production"
+)
+
```

Comment:
please add "app_name" and "version" like [this](https://github.com/openclimatefix/uk-pvnet-app/blob/main/src/pvnet_app/app.py#L49)